documentation/slate/source/includes/apis.md

```diff
@@ -20,6 +20,39 @@ The only arguments are the ones of [paging](#paging).
 | `coverage/{lon;lat}`                    | Information about a specific region |
 
 
+<a name="datasets"></a>Datasets
+-------------------------------
+
+Very simple endpoint providing the sets of data that are used in the given coverage.
```

Comment:
I think it is important to add some explanation on what is exactly a contributor and a dataset